app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None | str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Vaccination required to enter the cafe")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("Vaccine is expired. Need to renew it.")
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("A mask is required to enter the cafe")
```

Comment:
Code Style: Avoid using unnecessary `else`. After raising an exception, the function will exit, so there is no need to use `elif` here. You can use simple `if` statements.